cli/command/stack/cmd.go

```diff
@@ -23,7 +23,7 @@ type commonOptions struct {
 func NewStackCommand(dockerCli command.Cli) *cobra.Command {
 	var opts commonOptions
 	cmd := &cobra.Command{
-		Use:   "stack",
+		Use:   "stack [OPTIONS]",
```

Comment:
this output isn't actually used for commands that have subcommands, but good to have in case that situation changes in future